lib/dumper.rb

```diff
@@ -43,4 +44,8 @@ def category_id(_transaction)
   def normalize_iban(iban)
     iban.delete(' ')
   end
+
+  def cleared?(transaction)
+    date(transaction) < Time.now
```

Comment:
Maybe here we could use `Date.current` because the date method will return a date object (limitation of some dumpers, most banks don't return a timestamp..) and I think it reads a bit better to see a date comparing with a date. WDYT?